source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5868,9 +5869,17 @@ export class Container extends ContainerBase {
 
         swiperContainer.appendChild(swiperWrapper as HTMLElement);
 
+        cardLevelContainer.appendChild(swiperContainer);
+
         this.initializeSwiper(swiperContainer, nextElementDiv, prevElementDiv, pagination, displayProperties);
 
-        cardLevelContainer.appendChild(swiperContainer);
+        cardLevelContainer.onfocus = () => {
+            if (!this._isSwiperInitialized) {
+                this._isSwiperInitialized = true;
+                this._swiper?.destroy();
+                this.initializeSwiper(swiperContainer, nextElementDiv, prevElementDiv, pagination, displayProperties);
```

Comment:
why is this reinitialization needed?

Reply:
swiper's a11y init expects bullets, but the bullet is not generated; i believe it's because adaptivecard is not in the document. so on focus on the carousel, we check if the swiper is initialized. this way, if not we re-init, then the bullets will be there and allow a11y config to occur.